II.pass-tests/05-factorial/factorial.js

```diff
@@ -8,8 +8,9 @@
 // calculate and return the factorial of int
 // note: factorial of 0 is 1
 
-function factorial(int) {
-
+factorial = (int) => {
+    let f = [];
+  return  (int == 0 || int == 1) ? 1 : factorial(int-1) * int
```

Comment:
f variable is unused, also it's better not to use the ternary operator for clarity